pkg/models/plan_discussion.go

```diff
@@ -29,3 +30,18 @@ const (
 	DiscussionWaiting    DiscussionStatus = "WAITING_FOR_RESPONSE"
 	DiscussionUnAnswered DiscussionStatus = "UNANSWERED"
 )
+
+//IDiscussionRelation is an interface that represents models that are related to a discussion.
+type IDiscussionRelation interface {
+	GetDiscussionID() uuid.UUID
+}
+
+//DiscussionRelation is an embedded struct meant to satisify the IDiscussionRelation interface
+type DiscussionRelation struct {
+	DiscussionID uuid.UUID `json:"discussionID" db:"discussion_id"`
+}
+
+//GetDiscussionID returns DiscussionID
+func (d DiscussionRelation) GetDiscussionID() uuid.UUID {
+	return d.DiscussionID
```

Comment:
DiscussionRelation is bundled with `plan_discussion.go`, but the ModelPlanRelationhas its own file. Perhaps we should break this into a file as well, for consistency?

Reply:
I think that's a good call! Thanks!